### PR TITLE
Resolves #2: Added sidebar and navigation bar based on Material Schematics

### LINKED ISSUE #2: Sidebar and topbar based on Material Schematics
Use Angular material schematics to create a navigation that wraps the whole application

### PR DESCRIPTION
**Pull request by luiscbr92**
Pretty much the title. Here goes a screenshot with the result:
image

### CODE REVIEW

**Comment by Rsilnav** [comment]
Please merge develop back into the branc in order to get the latest changes before merging

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,534 +1,5 @@
...
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-
-<router-outlet></router-outlet>
\ No newline at end of file
+<div class="container">
```
Comment (Rsilnav) [inline src/app/app.component.html]: I do not think we need a container here, the router outlet would be in the `<app-navigation>` component, where it says 'Content here'.
Currently, If I'd write anything in this container I cannot see it.
I'd go for the other option.

File: src/app/navigation/navigation.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        type="button"
+        aria-label="Toggle sidenav"
+        mat-icon-button
+        (click)="drawer.toggle()"
+        *ngIf="isHandset$ | async">
+        <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
+      </button>
+      <span>etf</span>
+    </mat-toolbar>
+    <!-- Add Content Here -->
```
Comment (Rsilnav) [inline src/app/navigation/navigation.component.html]: Replace this line with `<router-outlet></router-outlet>`

File: src/app/app.component.html (hunk 1 above)
Comment (luiscbr92) [reply src/app/app.component.html]: Ah! I was expecting the content into the parent component instead of inside the navigation

**Comment by Rsilnav** [comment]
Looks good to me

**Pull request merged**